src/win/winapi.c

```diff
@@ -109,4 +114,16 @@ void uv_winapi_init() {
     pPowerRegisterSuspendResumeNotification = (sPowerRegisterSuspendResumeNotification)
       GetProcAddress(powrprof_module, "PowerRegisterSuspendResumeNotification");
   }
+
+  advapi32_module = GetModuleHandleA("advapi32.dll");
```

Comment:
This API is not exposed directly: https://msdn.microsoft.com/en-us/library/windows/desktop/aa387694.aspx

I tried another approach: https://boringssl.googlesource.com/boringssl/+/master/crypto/rand/windows.c#26 but ran into other troubles with redefined stuff. I plan to look into it once I have access to my Windows machine.